vcd/resource_vcd_vapp_firewall_rules.go

```diff
@@ -420,7 +425,7 @@ func listVappNetworksForImport(meta interface{}, orgName, vdcName, vappId string
 	vcdClient := meta.(*VCDClient)
 	_, vdc, err := vcdClient.GetOrgAndVdc(orgName, vdcName)
 	if err != nil {
-		return nil, fmt.Errorf("[vapp network firewall rules import] unable to find VDC %s: %s ", vdcName, err)
+		return nil, fmt.Errorf("[vapp network rules import] unable to find VDC %s: %s ", vdcName, err)
```

Comment:
As you're in `listVappNetworksForImport` func here, wouldn't it be more precise to say "[vapp network list] ..."?

Reply:
changed